apps/api/src/domain/models/photograph/queries/photograph-query-repository.interface.ts

```diff
@@ -13,7 +15,11 @@ export const PHOTOGRAPH_QUERY_REPOSITORY_TOKEN = 'PHOTOGRAPH_QUERY_REPOSITORY_TO
  * to serve as a constraint for the return of the query service and represents
  * a contract with the client.
  */
-export interface IPhotographQueryRepository extends IPublishable {
+export interface IPhotographQueryRepository
+    extends IPublishable,
+        IAccessible,
+        // TODO name other interfaces similar `IQueryRepositoryForXable`?
```

Comment:
That's a good idea. Also, shouldn't ICountable be here?

Reply:
I've added `ICountable`. I'm thinking we may have an `IResourceQueryRepo` at some point so we don't have to keep updating these lists. We'll never make a resource query repo that isn't publishable or countable, for example. Given that, let's hold off a bit longer.